api/test/shell/cli_test.sh

```diff
@@ -457,6 +457,23 @@ fi
 ./manager-api stop
 sleep 6
 
+# test manager-api output for bad data on etcd
+# make a dummy entry
+./etcd-v3.4.14-linux-amd64/etcdctl put /apisix/routes/unique1 "{\"id\":}"
```

Comment:
I think you could try to sleep 1 after put to etcd

Reply:
Still not working😶